setuptools/dist.py

```diff
@@ -25,6 +25,14 @@
 from .py36compat import Distribution_parse_config_files
 
 
+_skip_install_eggs = False
+
```

Comment:
So again I want to emphasize that this is an internal setuptools implementation detail that should not be used outside of setuptools.